topsort-endpoints.yml

```diff
@@ -317,6 +326,10 @@ components:
               minItems: 1
               items:
                 $ref: '#/components/schemas/PurchaseItem'
+            occurred_at:
```

Comment:
Isn't there just one of these at the ImpressionEvent level?

Reply:
Yes, the diff looks like this is on `Impression` but this is actually on `PurchaseEvent` if you expand the lines